internal/cli/install.go

```diff
@@ -359,7 +364,7 @@ func (c *InstallCommand) Flags() *flag.Sets {
 		f.StringVar(&flag.StringVar{
 			Name:    "context-create",
 			Target:  &c.contextName,
-			Default: fmt.Sprintf("install-%d", time.Now().Unix()),
+			Default: "install-timestamp",
```

Comment:
Why the sentinel value here, instead of removing the `Default` and adding to the `Usage` something like `"The default value is 'install-' with a timestamp suffix, ex: 'install-123456'"`

Reply:
Sure, I can do that!